.config/forest.dic

```diff
@@ -15,6 +16,7 @@ RPC
 SECP
 VM
 blockchain
+blockstore
```

Comment:
If we're talking about block stores in general then I'd put a space between the two words. If we're talking about a code concept then I'd write `BlockStore` or `Blockstore` or similar.